domains/transfer/Transfer.API/API/Claiming/Api/Models/ClaimSubject.cs

```diff
@@ -0,0 +1,5 @@
+using System;
+
+namespace API.Claiming.Api.Models;
+
+public record ClaimSubject(Guid SubjectId);
```

Comment:
I would prefer that the timestamp for when this entity was created was saved. It may be needed later and is not possible to recreate at a later point.

Reply:
Added CreatedAt and migrations